pylmod/tests/test_gradebook.py

```diff
@@ -805,13 +807,195 @@ def test_spreadsheet2gradebook_multi(self):
             ]
         )
 
+    @data(
+        (50, 0, True, 50),  # happy case
+        ('50', '0', True, 50),  # happy case with strings
+        ('', '0', True, 1),  # max_points is an invalid number
+        (None, '0', True, 1),  # max_points is missing
+        ('50', '1', True, 1),  # normalize is true
+        ('50', '', True, 1),  # normalize is an invalid number
+        ('50', None, True, 1),  # normalize is missing
+        ('50', '0', False, 1),  # use_max_points_column is False
+    )
+    @unpack
+    @httpretty.activate
+    def test_use_max_points_column(
+            self, max_points, normalize, use_max_points_column,
+            expected_max_points):
+        """
+        If use_max_points_column is True and normalize? is False,
+        use the max_points value.
+        """
+        new_assignment_id = 3
+        # Make sure that an error is raised if self.ASSIGNMENT_BODY
+        # is updated without updating this test too
+        for assignment in self.ASSIGNMENT_BODY['data']:
+            assert assignment['assignmentId'] != new_assignment_id
+
+        def handle_create(request, uri, headers):
+            """Assert that the create POST looks the way we expect it to"""
```

Comment:
why is this inside the test? if it should, please move it at the beginning of the test